config/locales/en_node_details.yml

```
@@ -50,6 +58,10 @@ en:
     texts:
       'yes': 'Yes'
       'no': 'No'
+      d_w_s: day / week / season
+      i_h_d: immediate / hour / day 
+      i_h_d_w: immediate / hour / day / week
+      i_h_d_w_s: immediate / hour / day / week / season
```

When I first saw these in a screenshot I found them pretty confusing. The reason being that the table already has entries like "€10 / full load hour" and "8 / year" where the "/" means "per": "€10 per full load hour" and "8 per year".

The result was that I had a hard time parsing what "immediate / hour / day" was supposed to mean. How would you feel about changing these to:

```suggestion
      d_w_s: daily, weekly, or seasonally
      i_h_d: immediately, hourly, or daily
      i_h_d_w: immediately, hourly, daily, or weekly
      i_h_d_w_s: immediately, hourly, daily, weekly, or seasonally
```

Ditto for the Dutch translations.

---

Good idea! I like it

---

I will update it